x-pack/plugins/apm/public/components/app/RumDashboard/CoreVitals/index.tsx

```diff
@@ -28,7 +29,7 @@ export function CoreVitals({ data, loading }: Props) {
       <EuiFlexItem>
         <CoreVitalItem
           title={LCP_LABEL}
-          value={lcp ? lcp + ' s' : '0'}
+          value={formatToSec(lcp, 'ms')}
```

Comment:
Is the output of this function going to be a string presented to the user, or is 'ms' an enumerable string?

If this will cause `ms` to show to the user make sure the output is translated.

Reply:
this is already done in another PR